editor/effects.js

```diff
@@ -19,9 +19,10 @@ export default {
 	REQUEST_POST_UPDATE( action, store ) {
 		const { dispatch } = store;
 		const { postId, edits } = action;
-		const isNew = ! postId;
 		const toSend = postId ? { id: postId, ...edits } : edits;
+		const isNew = ! postId;
 
+		dispatch( { type: 'CLEAR_POST_EDITS' } );
```

Comment:
Question: Should we wait until after the save has completed to clear edits? What if the save fails?